nifi-nar-bundles/nifi-accumulo-bundle/nifi-accumulo-processors/src/main/java/org/apache/nifi/accumulo/processors/ScanAccumulo.java

```diff
@@ -270,8 +281,9 @@ public void process(final InputStream in, final OutputStream out) throws IOExcep
                                         data.put("columnQualifier", key.getColumnQualifier().toString());
                                         data.put("columnVisibility", key.getColumnVisibility().toString());
                                         data.put("timestamp", key.getTimestamp());
+                                        data.put("value", kv.getValue().toString());
```

Comment:
Is it possible here for the value to be null? I don't know much about accumulo, but i would guess so. If so, we need to check if null and if so avoid calling `toString()` on the null reference

Reply:
To be on the safe side I've added the null check